### DISCUSSION TITLE
Can you update the script to search all drives instead of just C:

### DISCUSSION THREAD

**Discussion by qsub** [N-able/ScriptsAndAutomationPolicies#30] [Ideas]
As per the discussion topic, can you update the script to search all drives instead of just the C:

**Comment by githubtran** [comment]
Wrap the try-catch for robocopy into a foreach to search for all local drives

$DriveArray=@()
$DriveArray+=(Get-WmiObject -Class Win32_logicaldisk -Filter "DriveType = '3'").DeviceID
foreach ($Drive in $DriveArray) {
    $DriveVar="$Drive\"
    try {

          $robocopyexitcode = (start-process robocopy  -argumentlist "$driveVar c:\DOESNOTEXIST *.jar /S /XJ /L /FP /NS /NC /NDL /NJH /NJS /r:0 /w:0 /LOG:$env:temp\log4jfilescan.csv" -WindowStyle hidden -wait).exitcode
          if ($? -eq $True) {
              $robocopycsv = $true
              $log4jfilescan += import-csv "$env:temp\log4jfilescan.csv" -header FilePath -delimiter "|"       
              $log4jfilenames = $log4jfilescan
